pypeit/coadd3d.py

```diff
@@ -578,7 +588,9 @@ def set_default_scalecorr(self):
                 msgs.info("Loading default scale image for relative spectral illumination correction:" +
                           msgs.newline() + self.cubepar['scale_corr'])
                 try:
-                    spec2DObj = spec2dobj.Spec2DObj.from_file(self.cubepar['scale_corr'], self.detname)
+                    spec2DObj = spec2dobj.Spec2DObj.from_file(self.cubepar['scale_corr'],
```

Comment:
Given that `from_file` calls `io.fits_open` which checks if the file exists (and includes a few more try/except statements), I think this is a good point. It might be a good thing to fault, anyway. So, perhaps this code can just be changed to:

```
spec2DObj = spec2dobj.Spec2DObj.from_file(self.cubepar['scale_corr'],
                                          self.detname,
                                          chk_version=self.chk_version)
self.relScaleImgDef = spec2DObj.scaleimg
self.scalecorr_default = self.cubepar['scale_corr']
```

and the equivalent for other instances.

Reply:
For similar reasons as in `show_spec2d.py`, I'm not sure we want to remove the try block entirely.  But I think it would be good to specify that the except block is specific to datamodel errors.  I.e., does it make sense to replace `Exception` with `PypeItDataModelError`?

Reply:
Yes, I think that would be equally fine here. However, would we need to keep in the `Exception`, as well? For example:

```
try:
    BLAH
except PypeItDataModelError as e:
    BLAH
except Exception as e:
    BLAH
else:
    BLAH
```

Reply:
If possible, I'd rather we not use the base Exception class, unless we know that we should handle all other exceptions in the same way.  I.e., try/except blocks are easier to maintain/debug when they focus on specific known failure modes that we know how to handle.